app/lib/screens/study/onboarding/study_selection.dart

```diff
@@ -75,18 +91,32 @@ class StudySelectionScreen extends StatelessWidget {
                 ),
               ),
               Expanded(
-                child: RetryFutureBuilder<List<Study>>(
+                child: RetryFutureBuilder<ExtractionResult<Study>>(
                   tryFunction: () async => Study.publishedPublicStudies(),
-                  successBuilder: (BuildContext context, List<Study>? studies) {
+                  successBuilder: (BuildContext context, ExtractionResult<Study>? extractionResult) {
+                    final studies = extractionResult!.extracted;
+                    if (extractionResult is ExtractionFailedException<Study>) {
+                      debugPrint('${extractionResult.notExtracted.length} studies could not be extracted.');
+                      WidgetsBinding.instance.addPostFrameCallback((_) {
+                        ScaffoldMessenger.of(context).showSnackBar(
+                          SnackBar(
+                            content: Text(AppLocalizations.of(context)!.study_selection_hidden_studies),
+                          ),
+                        );
+                      });
```

Comment:
The snackbar is only shown for a limited amount of time and contains too much information. I would prefer displaying a proper warning to the user instead in form of a `MaterialBanner` between the headline and the `ListView`. The following should work:
```
MaterialBanner(
  leading: Icon(
    MdiIcons.exclamationThick,
    color: Colors.orange,
    size: 32,
  ),
  content: Text(
      AppLocalizations.of(context)!
          .study_selection_hidden_studies,
      style: Theme.of(context).textTheme.titleMedium),
  actions: [Container()],
  backgroundColor: Colors.yellow[100],
);
```

Adapted from [here](https://github.com/hpi-studyu/studyu/blob/d4bc7b5ee7494229bd09c50a6562fd9a53a922b1/app/lib/screens/study/onboarding/eligibility_screen.dart#L111).

Reply:
Done